### ISSUE TITLE
Add Files Duplicate command

### ISSUE THREAD

**Issue by JollyRoger8X** [files-community/Files#14135]
### What feature or improvement do you think would benefit Files?

Please consider adding a Files **Duplicate** command to duplicate a file in one step.

For instance, in macOS FInder, you can right-click a file and choose **Duplicate** to quickly create a duplicate "filename copy.ext" file. Or you can choose **File > Duplicate** with one or more files selected to duplicate them.

This is faster and easier than having to copy and paste the file(s).

### Requirements

As a user, I would liketo be able to duplicate selected file(s) with one quick action.

### Files Version

Preview 3.0.13.0

### Windows Version

10.0.19045.3324

**Comment by yaira2** [comment]
Thank you for the feedback, we'll track this as a discussion for now to gauge interest.

**Comment by yaira2** [comment]
@JollyRoger8X have you tried ctrl + c & ctrl + v? This is just as quick (if not quicker) than opening the context menu and clicking an option to "duplicate".

**Comment by JollyRoger8X** [comment]
Yes, of course. But that's literally double the time and effort of Control-D (Duplicate).

This is a very simple request to implement, and Copy & Paste would still be available, so I'm not sure resistance is really warranted.

**Comment by yaira2** [comment]
> As a reference, in macOS FInder, you can right-click a file and choose Duplicate to quickly create a duplicate filename copy.ext file. Or you can choose File > Duplicate with one or more files selected to duplicate them

We're not against the option but as explained above, the context menu isn't quicker than ctrl + c & ctrl + v. `ctrl` + `d` is already used for deleting items but once we're talking about keyboard shortcuts, it's only one more key press to release `c` and press `v` to paste the items.

**Comment by JollyRoger8X** [comment]
It seems more like you're just looking for excuses not to do it, but it's not a big deal either way. This has always been one in a long list of minor Windows irritations for me, and I guess it will remain so. Windows is the living embodiment of death by a thousand cuts either way. 🤣

**Comment by yaira2** [comment]
I hear how a keyboard shortcut could be useful which is why we're trying to determine how much interest there is in this option, but it also depends on whether there is already a solution that does what you're looking for.
